solc/CommandLineInterface.cpp

```diff
@@ -387,14 +387,12 @@ bool CommandLineInterface::readInputFiles()
 	{
 		if (!boost::filesystem::exists(m_fileReader.basePath()))
 		{
-			serr() << "Base path does not exist: " << m_fileReader.basePath() << endl;
-			return false;
+			BOOST_THROW_EXCEPTION(CommandLineValidationError() << errinfo_comment("Base path does not exist: "));
 		}
```

Comment:
These braces are no longer needed. Please remove them here and in other places. Making the code less verbose is one of the goals of this change after all.